Atomic/syscontainers.py

```diff
@@ -419,6 +420,8 @@ def _do_checkout(self, repo, name, img, upgrade, values, destination, unitfileou
                 key, val = i[:split], i[split+1:]
                 values[key] = val
 
+        if "UUID" not in values:
+            values["UUID"] = str(uuid.uuid4())
```

Comment:
it generates an UUID in the NNNNNNNN-NNNN-NNNN-NNNN-NNNNNNNNNNNN format that is the format expected by systemd for $container_uuid